pms/tests/test_pms_folio_invoice.py

```diff
@@ -121,7 +117,6 @@ def test_invoice_full_folio(self):
 
     def test_invoice_partial_folio_by_steps(self):
         # ARRANGE
```

Comment:
a short descriptive text of the task of the test can help a lot to understand what is being tested

Reply:
folio invoice test cases refactoring not included int thi PR, we will include them in another PR to be able to activate the changes of this